ibis-server/rust/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "wren-core-wrapper"
```

```suggestion
name = "wren-core-python"
```
Maybe `wren-core-python` better.

---

I will rename the name to `wren-modeling-py` to pair with `wren-modeling-rs`.